.vscode/launch.json

```diff
@@ -1,6 +1,14 @@
 {
     "version": "0.2.0",
     "configurations": [
+        {
+            "type": "node",
+            "request": "attach",
+            "name": "Attach by Process ID",
+            "processId": "${command:PickProcess}",
+            "restart": true,
+            "protocol": "inspector",
+        },
```

Comment:
Is this intentionally included?

Reply:
No that seems like my own debug configuration that was added. Ill fix it